wally-registry-backend/src/auth.rs

```diff
@@ -73,7 +121,40 @@ impl<'r> FromRequest<'r> for ReadAccess {
 }
 
 pub struct WriteAccess {
-    _dummy: i32,
+    github: Option<GithubInfo>,
+}
+
+impl WriteAccess {
+    pub fn github(&self) -> &Option<GithubInfo> {
+        &self.github
+    }
```

Comment:
Does this method need to exist now that `can_write_package` exists?

Reply:
Nope!